graphify-out/GRAPH_REPORT.md

```
@@ -1743,289 +1742,285 @@ Nodes (0):
 Cohesion: 1.0
 Nodes (1): kakao_login_test (patrol, skipped)
 
-### Community 271 - "Community 271"
+### Community 270 - "Community 270"
 Cohesion: 1.0
 Nodes (0): 
 
-### Community 272 - "Community 272"
+### Community 271 - "Community 271"
 Cohesion: 1.0
 Nodes (1): _StepIndicator / _Footer (wizard parts)
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside emphasis markers

(MD037, no-space-in-emphasis)